src/components/Search.jsx

```diff
@@ -2,9 +2,11 @@ import { useState } from "react";
 import style from "../components/Search.module.scss";
 import { useDebounce } from "../hooks/useDebounce";
 import PropTypes from "prop-types";
-
+import { useTheme } from "../hooks/themeUtils";
```

Comment:
Add an empty line between a component and imports.

Reply:
done